psl-clojure/template/resources/leiningen/new/psl_clojure/data/knows_obs.txt

```diff
@@ -0,0 +1,5 @@
+p1	p2
```

Comment:
I think this is dangerous to have the file named to the PSL conventions, but then include the header row.
Can we instead have these headers defined when the file is loaded.